src/ado2gh/Commands/GenerateScriptCommand.cs

```diff
@@ -179,6 +187,18 @@ public async Task Invoke(string githubOrg, string adoOrg, FileInfo output, bool
             return repos;
         }
 
+        private async Task GetTeamProjectRepos(AdoApi ado, Dictionary<string, IDictionary<string, IEnumerable<string>>> repos, string org, string teamProject)
```

Comment:
I'd suggest returning an IEnumerable<string> and adding it to the repos Dict in the caller instead of passing in repos here.